Ray.jl/src/runtime.jl

```diff
@@ -131,9 +134,20 @@ function task_executor(ray_function, ray_objects)
         io = IOBuffer(v)
         return deserialize(io)
     end
+
     arg_string = join(string.("::", typeof.(args)), ", ")
     @info "Calling $func($arg_string)"
-    return func(args...)
+    result = func(args...)
+
+    # TODO: remove - useful for now for debugging
+    @info "Result: $result"
+
+    # TODO: support multiple return values
+    buffer_data = Vector{UInt8}(sprint(serialize, result))
```

Comment:
is this better than printing to an IOBuffer and taking from it?

Reply:
Should be equivalent. Mainly this is just more terse.